Dockerfile

```diff
@@ -1,13 +1,22 @@
-FROM google/debian:wheezy
+FROM gcr.io/google_appengine/base
```

Comment:
We have debian, but it's still under gcr.io/google_appengine/debian. It never has patches applied either, so this one is safer to use. We couldn't push images to gcr.io/google/.